docs/examples/protein_alignment/2hhb.pdb

```diff
@@ -2663,7 +2663,7 @@ ATOM   2014  N   THR B 123      24.217  -7.708  -5.373  1.00 17.94           N
 ATOM   2015  CA  THR B 123      23.829  -7.414  -6.760  1.00 23.37           C  
 ATOM   2016  C   THR B 123      22.296  -7.446  -6.917  1.00 24.37           C  
 ATOM   2017  O   THR B 123      21.620  -7.945  -6.037  1.00 15.04           O  
-ATOM   2018  CB  THR B 123      24.443  -8.449  -7.684  1.00 16.11           C  
+ATOM   2020  CB  THR B 123      24.443  -8.449  -7.684  1.00 16.11           C  
```

Comment:
I don't think this `2018` should be changed to `2020`.

Reply:
Thank you for the careful reviewing. The problem has been fixed in 638d317.